package.json

```diff
@@ -36,6 +36,7 @@
     "scanAndFix": "dx-component-builder-sdk scanAndFix",
     "startStorybook": "start-storybook --port 6040 --quiet",
     "_comment_SDK_public_SDK_test_commands": "The following are public commands used for testing the SDK",
+    "test:functional": "jest --runInBand --detectOpenHandles --coverage",
     "test": "node ./scripts/playwright-message.js && playwright test --project=chromium MediaCo/portal MediaCo/embedded",
```

Comment:
Just a question. 

Should we run 'test:functional' from 'test' so that all tests run together? Or would that cause deployment issues?

Reply:
We are planning to get rid of e2e (playwright), then this can be moved to test directly.